pkg/account/domain/account.go

```diff
@@ -59,6 +80,53 @@ func NewAccountV2(
 	}}
 }
 
+func (a *AccountV2) Update(
```

Comment:
It appears that the `Update` function is utilized only in unit tests and not elsewhere.